### PR TITLE
Update bootstrapper to use the compiled version of KoreBuild

### PR DESCRIPTION
**Pull request by natemcmaster**
These are the changes required to update the bootstrapper. aspnet/KoreBuild#258

Resolves https://github.com/aspnet/AspNetCoreModule/issues/124

### CODE REVIEW

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj [hunk 1]
```diff
@@ -1,21 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
```
Comment (ryanbrandenburg) [inline test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:3]: Why the downgrade?

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+    <TargetFramework>netcoreapp1.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="1.0.0-msbuild3-final" />
+    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="1.0.*" />
```
Comment (ryanbrandenburg) [inline test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:6]: Looks like we changed the float on all of these, what were we targeting?

File: version.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<!-- This file may be overwritten by automation. -->
+<Project>
+  <PropertyGroup>
+    <KoreBuildChannel>dev</KoreBuildChannel>
```
Comment (ryanbrandenburg) [inline version.xml:4]: I guess we don't version this thing then? Or at least, not in the same ways?

**Review by ryanbrandenburg — APPROVED** [review]
Couple of questions, looks good in general.

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj (hunk 1 above)
Comment (natemcmaster) [reply test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:3]: The TFM didn't match the package versions. Packages were 1.0, tfm was 1.1

File: test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj (hunk 2 above)
Comment (natemcmaster) [reply test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj:6]: We want this project to be LTS for now. We don't care if this repo uses nightly aspnetcore.

File: version.xml (hunk 3 above)
Comment (natemcmaster) [reply version.xml:4]: Correct. this repo doesn't produce a managed nupkg.

**Pull request merged**